standalone/noPlugins_build.gradle

```diff
@@ -0,0 +1,175 @@
+// This is a sample gradle file for creating a stand-alone Java module that can be deployed in a LabKey server
+// instance.  This file assumes the module layout described in the LabKey documentation
+// (https://labkey.org/Documentation/wiki-page.view?name=moduleDirectoryStructures)
+// but, for simplicity, does not include JSP files that would be compiled and collected into a JSP jar file
+// or XSD files that would be transformed into XMLBeans objects and include in the main jar.
+//
+// You can refer to the gradlePlugins source code (https://github.com/LabKey/gradlePlugin) for more details on the
+// full build process implemented for modules within the LabKey source tree.  
+//
+// Also for simplicity, this example deliberately does not use the LabKey Gradle plugins, but if your module 
+// has JSPs or other client-side code and XMLBeans you may want to consider using these plugins.  You will
+// need to add a dependency on the plugins (see the settings.gradle file in LabKey Server's root enlistment)
+// and then apply the appropriate plugins, probably most simply the 'org.labkey.module' plugin, which, in turn,
+// will apply, for example, the JSP and NpmRun plugins, among others.
```

Comment:
Perhaps update the comment here to reflect that these things are illustrated in the `build.gradle` and `settings.gradle` files in this repo.